extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -299,6 +310,22 @@
   background-size: auto !important;
 }
 
+.i-amphtml-story-close-control {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="#FFFFFF"><path d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z"/></svg>') !important;
+}
+
+[desktop] .i-amphtml-story-close-control {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 32 32" fill="#FFFFFF"><path d="M25.3332 8.54663L23.4532 6.66663L15.9998 14.12L8.5465 6.66663L6.6665 8.54663L14.1198 16L6.6665 23.4533L8.5465 25.3333L15.9998 17.88L23.4532 25.3333L25.3332 23.4533L17.8798 16L25.3332 8.54663Z"/></svg>') !important;
+}
+
+.i-amphtml-story-skip-next {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 32 32" fill="#FFFFFF"><path d="M19.3333 16L8 24V8L19.3333 16ZM24 24H21.3333V8H24V24Z"/></svg>') !important;
+}
+
+.i-amphtml-story-skip-next[disabled] {
+  opacity: 0.3 !important;
+}
```

Comment:
Can you factorize to make it work with any `[disabled]` button?

https://github.com/ampproject/amphtml/blob/094908fb576384a602c31ac96411616bb1a8c73a/extensions/amp-story/1.0/amp-story-system-layer.css#L244-L247

Reply:
Done.